SECURITY.md

```diff
@@ -18,6 +18,14 @@ As Git for Windows bundles more than just Git (such as Bash, OpenSSL, OpenSSH, G
 
 Every Git for Windows version is tagged using a name that starts with the Git version on which it is based, with the suffix `.windows.<patchlevel>` appended. For example, Git for Windows v2.17.1' source code is tagged as [`v2.17.1.windows.1`](https://github.com/git-for-windows/git/releases/tag/v2.17.1.windows.1) (the patch level is always at least 1, given that Git for Windows always has patches on top of Git). Likewise, Git for Windows v2.17.1(2)' source code is tagged as [`v2.17.1.windows.2`](https://github.com/git-for-windows/git/releases/tag/v2.17.1.windows.2).
 
+## Release Candidates (rc) and Snapshot versions ('nighltlies')
+
+Git for Windows also releases versions that reflect the upstream release candidates, which contain the 'rc<n>' suffix to the regular git version, and before the 'windows' suffix. It should be noted that these rc version sort after their formal release, so appear to be newer to the updater software.
+
+(All releases)[https://github.com/git-for-windows/git/releases/] are listed via a link at the footer of the (Git for Windows)[https://gitforwindows.org/] home page.
```

Comment:
I think markdown links are `[formatted like this](https://help.github.com/en/articles/basic-writing-and-formatting-syntax#links)`. You did it the other way round.

Reply:
@bbolli Thanks. I saw the one error in the CI but without any real diagnostics, wee not that I could understand. I found [Is there any way to escape angle brackets](https://github.com/gollum/gollum/issues/713), so I've just force pushed that.

But yes, it does look like I'd also got the URL `[]()` the wrong way around

Reply:
@bbolli Does the content, headings, etc. read OK?